ecs-init/cache/cache.go

```diff
@@ -91,6 +104,16 @@ func (d *Downloader) getRegion() string {
 	return d.region
 }
 
+// getBucketRegion returns a region that contains the agent's bucket
+func (d *Downloader) getBucketRegion() string {
+	region := d.getRegion()
+	if config.IsSupportedRegion(region) {
+		return region
+	}
+
+	return config.DefaultRegionName
```

Comment:
Can you add a log line here stating that the supported region could not be determined?